gensrc/proto/internal_service.proto

```diff
@@ -111,6 +111,15 @@ message PTabletWriterOpenResult {
     repeated PTabletLoadRowsetInfo tablet_load_rowset_num_infos = 2;
 };
 
+message PFinalizeCloseLoadChannelRequest {
+    required PUniqueId id = 1;
```

Comment:
all member should be optional?

Reply:
sorry, some commits were brought up by mistake. fixed.